k8sutils/agesecret.go

```diff
@@ -22,6 +22,9 @@ var (
 	unwantedAnnotations = []string{
 		"kubectl.kubernetes.io/last-applied-configuration",
 	}
+	unwantedLabels = []string{
```

Comment:
Why not make this a constant (or a function that returns a constant) and use it both here and in tests?